api/src/main/java/io/druid/utils/Runnables.java

```diff
@@ -19,14 +19,15 @@
 
 package io.druid.utils;
 
+import io.druid.guice.annotations.PublicApi;
+
 /**
  */
+@PublicApi
 public class Runnables
```

Comment:
This one is probably here since it's used by FileIteratingFirehose which is in druid-api.